src/assets/scripts/groepen/Civikaartje.jsx

```diff
@@ -0,0 +1,40 @@
+var React = require("react");
+var $ = require("jquery");
+var _ = require("underscore");
+
+class Civikaartje extends React.Component {
+
+  constructor(props, context) {
+    super(props);
+    this.profiel = props.profiel;
+    this.context = context;
+
+    this.go = this.go.bind(this)
+  }
+
+  go(event) {
+    this.context.router.transitionTo('profiel-detail', {uid: this.profiel.uid});
```

Comment:
It's easier to create a `<Link to="profiel-detail" params={{uid: this.profiel.uid}}> `in the render